src/scss/base/validation.scss

```diff
@@ -0,0 +1,6 @@
+input {
+    &.error, &.error:hover, &.error:active, &.error:focus {
+        border: 1px solid #eb4034 !important;
```

Comment:
is `!important` required here?

Reply:
@squallstar 
Yes, there is a color picker library class which is rewrites this rule. Therefore we need to use `!important`